app/filters/filters.js

```diff
@@ -345,4 +345,15 @@ angular.module('portainer.filters')
   return function (createdBy) {
 	  return createdBy.replace('/bin/sh -c #(nop) ', '').replace('/bin/sh -c ', 'RUN ');
   };
-});
+})
+.filter('trimshasum', function () {
+  'use strict';
+  return function (imageName) {
+    if (imageName) {
+      var name = imageName.split('sha256:')[0];
+      return name ? name : imageName.substring(7, 19);
```

Comment:
I'd go with

```
return ( imageName[0].substring(0,7) === 'sha256:' ) ? imageName[0].substring(7, 19) : imageName[0];
```

because I feel that `split` is not required if we know that we want to (always) check the first seven chars. But it's a matter of style, so let's wait for @deviantony to reply.